### ISSUE TITLE
Lucene highlighter failed when filter by text with `{}` symbols

### ISSUE THREAD

**Issue by sfairat15** [microsoft/K2Bridge#136]
Lucene highlighter failed when filter by text with `{ }` symbols.

I am trying to use filter and it filters as expected:
```
{
  "query": {
    "match_phrase": {
      "Payload.@mt": "Lucene highlighter is enabled: {isHighlight}"
    }
  }
}
```

But Kibana did not highlight text and I can see next errors at K2Bridge log:
```
  | Payload.@l | Error
  | Payload.@mt | Failure creating highlighters for {value}
  | Payload.@t | Jan 19, 2022 @ 15:04:25.122
  | Payload.@x | Lucene.Net.QueryParsers.ParseException: Cannot parse 'Lucene highlighter is enabled: {isHighlight}': Encountered " "}" "} "" at line 1, column 43. Was expecting one of:     "TO" ...     <RANGEEX_QUOTED> ...     <RANGEEX_GOOP> ...       ---> Lucene.Net.QueryParsers.ParseException: Encountered " "}" "} "" at line 1, column 43. Was expecting one of:     "TO" ...     <RANGEEX_QUOTED> ...     <RANGEEX_GOOP> ...         at Lucene.Net.QueryParsers.QueryParser.Jj_consume_token(Int32 kind)    at Lucene.Net.QueryParsers.QueryParser.Term(String field)    at Lucene.Net.QueryParsers.QueryParser.Clause(String field)    at Lucene.Net.QueryParsers.QueryParser.Query(String field)    at Lucene.Net.QueryParsers.QueryParser.Parse(String query)    --- End of inner exception stack trace ---    at Lucene.Net.QueryParsers.QueryParser.Parse(String query)    at K2Bridge.KustoDAL.LuceneHighlighter.MakeValueHighlighter(QueryParser parser, String value, String highlightPreTag, String highlightPostTag) in /app/K2Bridge/KustoDAL/LuceneHighlighter.cs:line 171
  | Payload.ActionId | af9a55f3-a30f-4f19-a348-674aa037d0c3
  | Payload.ActionName | K2Bridge.Controllers.QueryController.SingleSearchAsync (K2Bridge)
  | Payload.ConnectionId | 0HMER3KCK6TR2
  | Payload.CorrelationId | 3f885b0f-be58-4021-be3d-02360c800848
  | Payload.RequestId | 0HMER3KCK6TR2:00000329
  | Payload.RequestPath | /log:InfraKustokibana/_search
  | Payload.SourceContext | K2Bridge.KustoDAL.KustoResponseParser


   ```

**Comment by sfairat15** [comment]
Reproduced on dev-vnext branch

**Comment by AsafMah** [comment]
Solved with #143
